schema/crdb/dbinit.sql

```diff
@@ -213,7 +223,7 @@ CREATE TABLE IF NOT EXISTS omicron.public.switch (
     time_created TIMESTAMPTZ NOT NULL,
     time_modified TIMESTAMPTZ NOT NULL,
     time_deleted TIMESTAMPTZ,
-    rcgen INT NOT NULL,
+    rcgen INT8 NOT NULL,
```

Comment:
At the tail end of this file, there is a `14.0.0` that needs to be updated to `15.0.0` for the tests to pass

Reply:
Oh I should revert the `rcgen` change, it's not relevant (though maybe we do want to do that as a blanket change)

Reply:
Done.